src/fill-in-the-blank/components/runtime.tsx

```diff
@@ -46,7 +49,10 @@ export const insertInputs = (prompt: string, blanks: IBlankDef[], userResponses:
   return result;
 };
 
-export const Runtime: React.FC<IProps> = ({ authoredState, interactiveState, setInteractiveState, report }) => {
+export const Runtime: React.FC<IProps> = ({ authoredState, interactiveState, setInteractiveState, setNavigation, report }) => {
+  const submitEnabled = (interactiveState?.blanks || []).length > 0;
```

Comment:
This might be more clear if the variable name and the one passed to useRequiredQuestion was `isAnswered` or `hasAnswer`.  I was confused because there are many things that can change the submitButton, so using submitEnabled seemed to conflict with the other logic.